src/components/menuItemPropTypes.js

```diff
@@ -68,7 +69,7 @@ const propTypes = {
     subtitle: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
 
     /** Flag to choose between avatar image or an icon */
-    iconType: PropTypes.oneOf([CONST.ICON_TYPE_AVATAR, CONST.ICON_TYPE_ICON]),
+    iconType: PropTypes.oneOf([CONST.ICON_TYPE_AVATAR, CONST.ICON_TYPE_ICON, CONST.ICON_TYPE_WORKSPACE]),
```

Comment:
nab, or probably an overkill. Do we want to create this as a prop in `avatarPropTypes`? and use that instead of importing this everywhere?

Reply:
I think no, but only because we can have different iconTypes:
MenuItem can take in:
[CONST.ICON_TYPE_AVATAR, CONST.ICON_TYPE_ICON, CONST.ICON_TYPE_WORKSPACE]
Whereas Avatars take in:
[CONST.ICON_TYPE_AVATAR, CONST.ICON_TYPE_WORKSPACE]

Although one option would be that we could make separate propTypes for these. Like `MenuItemIconTypes` and `AvatarIconTypes`. What are your thoughts given that?